packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyView/FoldersTableControl.tsx

```diff
@@ -18,10 +19,11 @@ export const { useFoldersTable, FoldersTableProvider } = createContextUtilities(
 );
 
 export const FoldersTableControl = (): React.JSX.Element => {
-  const { folders, onSelect } = useFoldersTable();
+  const { folders, isProcessing, onSelect } = useFoldersTable();
 
   const tableData = getDestinationPickerTableData({
     folders,
+    isLoading: !!isProcessing,
```

Comment:
We shouldn't have to feed `isLoading` to the DataTable via `tableData`

Reply:
Updated